src/Components/Publishing/Sections/FullscreenViewer/Slide.tsx

```diff
@@ -1,20 +1,24 @@
 import { omit } from "lodash"
 import * as React from "react"
-import styled from "styled-components"
+import styled, { StyledFunction } from "styled-components"
 import { resize } from "../../../../Utils/resizer"
 import { pMedia } from "../../../Helpers"
 import { Fonts } from "../../Fonts"
 import { Caption } from "./Caption"
 
+interface ImageProps {
+  src?: string
```

Comment:
Is this really optional? I don’t actually know, but assume that `background: url()` would be illegal?